src/content/en/fundamentals/performance/rail.md

```diff
@@ -78,18 +83,24 @@ For actions that take longer than 500ms to complete, always provide feedback.
 
 Success: Respond to user's touchmoves and scrolling in under 16ms.
 
-## Animation: render frames every 16ms
+## Animation: execute animation code in under 10ms
 
 Animations aren't trivial actions that web apps can opt into.
 For example, scrolling and touchmoves are types of animation.
```

Comment:
I don't understand the above. Animations sometimes are trivial things developers trigger (CSS animations for example). 

Should this be "Animations aren't just trivial..."?